Clear/main.py

```diff
@@ -0,0 +1,337 @@
+#! /usr/bin/python -u
```

Comment:
I understand that you've named this program `Clear` with a capital "C" to distinguish it from the standard Linux `clear` command. I feel as though it would be better off with a different, all-lowercase name to avoid confusion. Suggestions?

Reply:
Alternative is to alias "clear" with some script. In that script, if clear does not have arguments, invoke system clear command, otherwise invoke corresponding sonic clear command.

Reply:
maybe "clearcmd" ?

Reply:
The best way to address this is by having a small script with name "clear" installed under /usr/local/bin/ that redirects user input to system clear command (/usr/bin/clear) & SONiC Clear commands (/usr/bin/Clear). 
The user will not see the upper case Clear & the redirection will transparent to the user.
Since, that script change has go to sonic-buildimage repo, I can commit that change in a separate pull.

Reply:
all cmd should begin with lower case, that is linux

Reply:
can we /usr/bin/clear to /usr/bin/clear_screen

then, when we do 'clear screen', we invoke this /usr/bin/clear_screen

Reply:
This isn't a backwards compatible suggestion if we ever decide to change in the future the cli framework. The Linux cmds shouldn't touched.

Reply:
if there is no argument, we can still call linux "clear", there is no backwards compatibility issue.

Reply:
/usr/bin/clear is the linux clear screen command. It should not be replaced and it should not be renamed.

Reply:
The only issue I have with Ravi's solution of a /usr/local/bin/clear script is that it will make the SONiC `clear` command behave inconsistently with the other CLI commands. For instance, calling `show` or `config` with no arguments will present the user with the usage statement, but calling `clear` with no arguments will clear the user's screen.

Reply:
It is unfortunate that sonic presents the user with usage statement when show or config are called with no arguments. Sounds like that decision needs to be revisited. Nevertheless, we need to move forward with this commit. I agree with the point you made that we should come up with a better way than a capitalized command. Sort of introducing a cli shell script for all sonic commands (show, clear, config), this is the next best thing which is not interfering with the linux clear command in any way.